benchmarks/README.md

```diff
@@ -25,19 +25,39 @@ We analyzed the dataset using [quantms workflow](https://github.com/bigbio/quant
 
 In summary, both datasets were search with three search engines SAGE, COMET and MSGF+ and the results were combined with ConsesusID and PSMs and proteins were filtered with a 1% FDR. The quantification was performed with the quantms workflow using the ibaq method. Some general statistics: 
 
-- TMT: X samples, X proteins, X peptides, X features, X PSMs. 
-- LFQ: X samples, X proteins, X peptides, X features, X PSMs.
+- TMT: 11 samples, 9423 proteins, 77439 peptides, 1409771 features, 139891 PSMs. 
+- LFQ: 11 samples, 8213 proteins, 54939 peptides, 505906 features, 533910 PSMs.
 
 #### Coefficient of Variation (CV)
 
+<img src='../data/benchmark/cv.png' style="margin-left:200px;width:800px;height:600px;"></img>
+
 #### Correlation between TMT and LFQ samples
 
+<img src='../data/benchmark/PXD007683-TMTvsLFQ-density.png' style="margin-left:200px;width:800px;height:600px;"></img>
```

Comment:
@zprobot as I mentioned before in private chat; it would be great to have this plots in three different categories: 
- Previous [paper figures](https://analyticalsciencejournals.onlinelibrary.wiley.com/doi/10.1002/pmic.202300188)
- Figures generated with the main methods to compute ibaq in the current version `quantile`, `median`, `median-cov`. 

What do you think?